pkg/costmodel/router.go

```diff
@@ -1004,59 +971,40 @@ func Initialize(additionalConfigWatchers ...ConfigWatchers) *Accesses {
 
 	timeout := 120 * time.Second
 	keepAlive := 120 * time.Second
-	scrapeInterval, _ := time.ParseDuration("1m")
```

Comment:
This is... interesting. Any idea why it was like this before? Legacy from parsing a string from a config or Prom?

Reply:
I think it was written in a hurry or at the end of the day. Either way, I was happy to improve it :)